ext/standard/levenshtein.c

```diff
@@ -123,7 +123,7 @@ PHP_FUNCTION(levenshtein)
 	}
 
 	if (distance < 0 && /* TODO */ ZEND_NUM_ARGS() != 3) {
-		php_error_docref(NULL, E_WARNING, "Argument string(s) too long");
+		zend_value_error("Argument string(s) too long");
```

Comment:
Code may want to handle this condition, as I don't think the limit is exposed (and it's a low limit -- this is not a 2G string condition).